src/components/UI/GeneralTooltip/Tooltip.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import * as tooltip from '@material-ui/core/Tooltip';
```

Comment:
Can we follow the convention used here: https://github.com/glific/glific-frontend/blob/master/src/components/UI/Form/Button/Button.tsx